Gemfile

```diff
@@ -1,6 +1,9 @@
 source "https://rubygems.org"
 
-git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 # Specify your gem's dependencies in spire.gemspec
 gemspec
+
+# lower rails version
+gem "rails", "~> 4.0.0"
```

Comment:
@caseyli I went lower with rails 4 but can put this to whatever we want

Reply:
@7chris71000 so I like this functionality, but at the same time I don't want to restrict non-rails programs from using this gem. Would you mind researching how to make this optional? "Like if Rails is installed, run the generator. If Rails isn't installed, don't run it." I think that would be the best option.